sk.c

```diff
@@ -360,22 +368,22 @@ int sk_receive(int fd, void *buf, int buflen,
 	if (addr)
 		addr->len = msg.msg_namelen;
 
-	if (!ts) {
```

Comment:
Should probably be left in place, or split out to a separate patch with its own justification